utils/tests_fetcher.py

```diff
@@ -246,47 +258,55 @@ def create_reverse_dependency_map():
 # Any module file that has a test name which can't be inferred automatically from its name should go here. A better
 # approach is to (re-)name the test file accordingly, and second best to add the correspondence map here.
 SPECIAL_MODULE_TO_TEST_MAP = {
-    "commands/add_new_model_like.py": "test_add_new_model_like.py",
+    "commands/add_new_model_like.py": "utils/test_add_new_model_like.py",
     "configuration_utils.py": "test_configuration_common.py",
-    "convert_graph_to_onnx.py": "test_onnx.py",
-    "data/data_collator.py": "test_data_collator.py",
+    "convert_graph_to_onnx.py": "onnx/test_onnx.py",
+    "data/data_collator.py": "trainer/test_data_collator.py",
     "deepspeed.py": "deepspeed/",
     "feature_extraction_sequence_utils.py": "test_sequence_feature_extraction_common.py",
     "feature_extraction_utils.py": "test_feature_extraction_common.py",
-    "file_utils.py": ["test_file_utils.py", "test_model_output.py"],
-    "modelcard.py": "test_model_card.py",
+    "file_utils.py": ["utils/test_file_utils.py", "utils/test_model_output.py"],
+    "modelcard.py": "utils/test_model_card.py",
     "modeling_flax_utils.py": "test_modeling_flax_common.py",
-    "modeling_tf_utils.py": ["test_modeling_tf_common.py", "test_modeling_tf_core.py"],
-    "modeling_utils.py": ["test_modeling_common.py", "test_offline.py"],
-    "models/auto/modeling_auto.py": ["test_modeling_auto.py", "test_modeling_tf_pytorch.py", "test_modeling_bort.py"],
-    "models/auto/modeling_flax_auto.py": "test_modeling_flax_auto.py",
+    "modeling_tf_utils.py": ["test_modeling_tf_common.py", "utils/test_modeling_tf_core.py"],
+    "modeling_utils.py": ["test_modeling_common.py", "utils/test_offline.py"],
+    "models/auto/modeling_auto.py": [
+        "auto/test_modeling_auto.py",
+        "auto/test_modeling_tf_pytorch.py",
+        "bort/test_modeling_bort.py",
+    ],
+    "models/auto/modeling_flax_auto.py": "auto/test_modeling_flax_auto.py",
     "models/auto/modeling_tf_auto.py": [
-        "test_modeling_tf_auto.py",
-        "test_modeling_tf_pytorch.py",
-        "test_modeling_tf_bort.py",
+        "auto/test_modeling_tf_auto.py",
+        "auto/test_modeling_tf_pytorch.py",
+        "bort/test_modeling_tf_bort.py",
     ],
-    "models/blenderbot_small/tokenization_blenderbot_small.py": "test_tokenization_small_blenderbot.py",
-    "models/blenderbot_small/tokenization_blenderbot_small_fast.py": "test_tokenization_small_blenderbot.py",
```

Comment:
Those two lines look removed, is it intentional?

Reply:
Yes, intentional! The files have been renamed to be coherent with the tokenization files and folder name.

Reply:
Thank you so much for that!